File: apps/vinculacion/app/assets/javascripts/vinculacion/models/empleado.js
App.Empleado = DS.Model.extend({
  nombre:            DS.attr('string'),
  apellido_paterno:  DS.attr('string'),
  apellido_materno:  DS.attr('string'),
  email:             DS.attr('string'),
  telefono:          DS.attr('string'),
  nombre_completo:   DS.attr('string'),
  avatar_url:   DS.attr('string'),
  mailto:  function() {
    return 'mailto:' + this.get('email');
  }.property('email')
  //sede:              DS.belongsTo('sede'),
  //departamento:      DS.belongsTo('departamento'),
  //empleado:          DS.belongsTo('empleado'),
});

File: apps/proyectos/app/models/proyectos/empleado.rb
module Proyectos
  class Empleado < ActiveRecord::Base

    self.table_name = "rh_empleados"

    belongs_to :departamento

    def nombre_completo
      "#{nombre} #{apellido_paterno} #{apellido_materno}"
    end

    def avatar_url
      username = email.split(/@/)[0]
      "http://cimav.edu.mx/foto/#{username}/64"
    end

  end
end
